langchain/document_loaders/yuqueloader.py

```diff
@@ -0,0 +1,92 @@
+from typing import List, Tuple, Dict, Union
+
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseLoader
+
+
+class YUQUELoader(BaseLoader):
```

Comment:
Is there a reason to use all capital for the Class name (`YUQUELoader`)? 
`YuqueLoader` would be a better fit for the standard naming convention.